src/components/ButtonSubmit/index.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 const ButtonSubmit = ({ label }) => {
   return (
     <div className="button-submit">
```

Comment:
how bout naming this group is `form-submit` with element
`form-submit__button--white`?

Reply:
IMO, as this component is for a button it should be `button-submit`.

Reply:
How about button-wrapper? since you're gonna style the `div` not the `button` so, `button-submit` is not correct IMO

Reply:
As I also named the component `button-submit` which is confusing as well, I named everything to `FormButton`. It's better now I think.